templates/home/index.json-ld

```diff
@@ -54,5 +64,30 @@
 				"path": "reports/pre-a11y-1-fr.html"
 			}
 		]
-	}
+	},
```

Comment:
There is a documentation page already, called "api-en.html". Please either keep that one or fully replace the documentation by doing the complete index.json-ld. We cannot have two documentation pages.